### DISCUSSION TITLE
curl requires .... to build ?

### DISCUSSION THREAD

**Discussion by lbdyck** [zopencommunity/meta#194] [Porting]
Why so many build dependencies - and are git, tag, gzip, ... really required for the build?

![image](https://user-images.githubusercontent.com/42328411/219673088-98e343f1-b0c5-4ec5-8413-97480d0309a6.png)

**Comment by IgorTodorovskiIBM** [comment]
When we say "Build" we really mean Build/Test/Install or all of the dependencies needed to be able run `zopen build`, which uses git apply, etc. Maybe "zopen build dependencies" is a better term.

**Comment by lbdyck** [comment]
So those build dependencies are not specific to curl. Perhaps only the curl build dependencies should be mentioned in regards to curl. ??

**Reply by MikeFultonDev** [reply]
@lbdyck i guess the question is what would you do differently? If you use the community tool to build curl, you really do need all that stuff. If you want to roll your own build, then the list is smaller. I'm torn between 'cleaning up' the list or leaving it alone since I'm not sure how people building it will get value from the smaller list?

**Reply by lbdyck** [reply]
Igor's explanation makes sense to me - build is more than just build.
